server/src/controllers/common.es6

```diff
@@ -2,10 +2,67 @@
 
 const NoncommercialApplication = require('../models/noncommercial-application.es6');
 const TempOutfitterApplication = require('../models/tempoutfitter-application.es6');
+const util = require('../util.es6');
 
 const commonControllers = {};
```

Comment:
I wonder if we want to rename this to `specialUseCommonControllers` as they may be common across the two permit types now, but will likely be not used for the others.